src/lib.rs

```diff
@@ -106,27 +106,8 @@ pub use crate::splitting::{HyphenSplitter, NoHyphenation, WordSplitter};
 
 pub mod core;
 
-/// Options for wrapping and filling text. Used with the [`wrap`] and
-/// [`fill`] functions.
-///
-/// [`wrap`]: fn.wrap.html
-/// [`fill`]: fn.fill.html
-pub trait WrapOptions {
-    /// The width in columns at which the text will be wrapped.
-    fn width(&self) -> usize;
-    /// Indentation used for the first line of output.
-    fn initial_indent(&self) -> &str;
-    /// Indentation used for subsequent lines of output.
-    fn subsequent_indent(&self) -> &str;
-    /// Allow long words to be broken if they cannot fit on a line.
-    /// When set to `false`, some lines may be longer than `width`.
-    fn break_words(&self) -> bool;
-    /// Find indices where `word` can be split.
-    fn split_points(&self, word: &str) -> Vec<usize>;
-}
-
 /// Holds settings for wrapping and filling text.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Copy)]
```

Comment:
Is this related somehow? As far as i recall, the `Standard` type from the `hyphenation` crate is not `Copy`, so I guess the derived copy implementation won't work then — is there then a reason to have it?

Reply:
The reasoning was that it allows `Config` to be `Copy` in many places, which avoids users having to pass references to it when a copy would be fine.

Reply:
We'll still need to treat it as non-`Copy` internally, since we cannot rely on `Copy` being derived. This makes me think that it should at least go in a separate PR. I'm still a bit wary of offering one API for users who use a `Copy` type as the `WordSplitter` and a different API for other users — it might be surprising to people is their code stops compiling with the `hyphenation` feature. On the other hand, they might never use that feature and then it's indeed nicer for people if they can just copy around their `Options` as they like.

So yeah, please spin this off into a separate PR which we can then merge in parallel with this.